### PR TITLE
5x/cr 1306 comfort update show informative message before upgrading to ls6

### PR DESCRIPTION
**Pull request by ptelu**

### CODE REVIEW

File: application/views/admin/update/updater/steps/_change_log.php [hunk 1]
```diff
@@ -3,72 +3,73 @@
...
-?>
-
-
-
-<h3 class="maintitle"><?php eT('Change log'); ?></h3>
 
+?>
 
+<?php if (isset($destinationMajorVersion)) : ?>
+    <h3 class="maintitle">Welcome to LimeSurvey <?= $destinationMajorVersion ?>!</h3>
```
Comment (twilligls) [inline application/views/admin/update/updater/steps/_change_log.php:18]: no translation?!

File: application/config/config-defaults.php [hunk 2]
```diff
@@ -729,6 +729,8 @@
 $config['uploaddir']                = $config['rootdir'] . DIRECTORY_SEPARATOR . "upload";
 $config['standardthemerootdir']     = $config['rootdir'] . DIRECTORY_SEPARATOR . "themes" . DIRECTORY_SEPARATOR . "survey"; // The directory path of the standard themes
 $config['publicstylepath']          = $config['rootdir'] . DIRECTORY_SEPARATOR . $config['publicstyle'];
+$config['views']                    = $config['rootdir'] . DIRECTORY_SEPARATOR . 'application' . DIRECTORY_SEPARATOR . "views";
+$config['adminviews']               = $config['views']    . DIRECTORY_SEPARATOR . "admin";
```
Comment (twilligls) [inline application/config/config-defaults.php:733]: Are those entries still needed? Doesn't hurt when they stay though :)

**Review by twilligls — CHANGES_REQUESTED** [review]

File: application/config/version.php [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
 $config['versionnumber'] = '5.6.35';
 $config['dbversionnumber'] = 498;
-$config['buildnumber'] = '';
+$config['buildnumber'] = '230808';
```
[the remark this answers is not in the document]
Comment (ptelu) [reply application/config/version.php:16]: its already gone :)

File: application/config/config-defaults.php (hunk 2 above)
Comment (ptelu) [reply application/config/config-defaults.php:733]: yep thats what i thought

**Pull request merged**